src/c++/perf_analyzer/perf_analyzer.cc

```diff
@@ -481,9 +483,10 @@ Usage(char** argv, const std::string& msg = std::string())
       << FormatMessage(
              " --stability-percentage (-s): Indicates the allowed variation in "
              "latency measurements when determining if a result is stable. The "
-             "measurement is considered as stable if the recent 3 measurements "
-             "are within +/- (stability percentage)% of their average in terms "
-             "of both infer per second and latency. Default is 10(%).",
+             "measurement is considered as stable if the max and min of the "
```

Comment:
Let's be precise. Your statement of them being within x% of each other isn't clear. You are calculating if the max is within x% of the min.  That is different than the min being within x% of the max.

Reply:
Updated to explain max/min